www/includes/easyparliament/templates/html/mp/profile.php

```diff
@@ -313,3 +306,18 @@
         </div>
     </div>
 </div>
+
+<div id="whereNextModal" class="reveal-modal" data-reveal aria-labelledby="modalTitle" aria-hidden="true" role="dialog">
+    <h2 id="modalTitle">Where to next?</h2>
+    <p class="lead">Do you know what would you like to do with the information on this page?</p>
+    <ul>
+        <li><a id="research-qual2-bucket1-wherenext" href="/action/where-next">I&rsquo;m not sure &mdash; Click here for suggestions on what to do next</a></li>
+        <li><a data-research-qual2-bucket1-linkname="write-mp">I&rsquo;m going to write to an MP</a></li>
+        <li><a data-research-qual2-bucket1-linkname="social-share">I&rsquo;m going to share this information on social media</a></li>
+        <li><a data-research-qual2-bucket1-linkname="work">I&rsquo;m going to use this information for work purposes</a></li>
+        <li><a data-research-qual2-bucket1-linkname="browsing">I don&rsquo;t need to do anything with this information &mdash; I was just browsing</a></li>
+    </ul>
+</div>
+
+<script src="<?= cache_version('js/jquery.storageapi.min.js') ?>"></script>
+<script src="<?= cache_version('js/qual2.js') ?>"></script>
```

Comment:
Given this is repeated in two places could it not be pulled out into an include?